apps/shared/avifpng.h

```diff
@@ -11,7 +11,12 @@ extern "C" {
 #endif
 
 // if (requestedDepth == 0), do best-fit
-avifBool avifPNGRead(const char * inputFilename, avifImage * avif, avifPixelFormat requestedFormat, uint32_t requestedDepth, uint32_t * outPNGDepth);
+avifBool avifPNGRead(const char * inputFilename,
+                     avifImage * avif,
+                     avifPixelFormat requestedFormat,
+                     uint32_t requestedDepth,
+                     uint32_t * outPNGDepth,
+                     avifRGBToYUVFlags flags);
```

Comment:
Done for avifPNGRead and avifReadImage. I left avifInputReadImage as is since it was less clear where to put the flags and it's a function internal to avifenc.c anyway.